cypress/integration/vote_for_card_spec.js

```diff
@@ -1,11 +1,18 @@
 describe('Vote for a card', () => {
     beforeEach(() => {
         cy.login('unicorn');
+        cy.createRoom();
         cy.request(`http://localhost:12346/connect?url=${encodeURIComponent(Cypress.config().baseUrl)}`)
         .then(() => {
-            cy.startGame();
-            cy.request(`http://localhost:12346/playCardWord?url=${encodeURIComponent(Cypress.config().baseUrl)}`)
-            .then(() => cy.playCard());
+            cy.get('[data-cy="room-title"]').then(($title) => {
```

Comment:
Does it give it a random title?

Reply:
Nope, title is Room: X, where X is the room id

Reply:
Oh okay, so do you already know what the room id will be? Cause doing this split thing seems weird

Reply:
No I don't, what if we create more than one room